src/Solvers/search.jl

```diff
@@ -103,10 +118,20 @@ function check_cone_points(
         Cones.load_point(cone_k, cand.primal_views[k], irtmu)
         Cones.load_dual_point(cone_k, cand.dual_views[k])
         Cones.reset_data(cone_k)
-        in_nbhd_k = (Cones.is_feas(cone_k) && Cones.is_dual_feas(cone_k) &&
-            Cones.in_neighborhood(cone_k, rtmu, max_nbhd))
-        step_searcher.cone_times[k] = time() - start_time
-        in_nbhd_k || return false
+
+        in_prox_k = false
+        if Cones.is_feas(cone_k) && Cones.is_dual_feas(cone_k) &&
+            Cones.check_numerics(cone_k)
+            prox_k = Cones.get_proximity(cone_k, rtmu, use_sum_prox)
+            if !isnan(prox_k) && (prox_k < prox_bound)
```

Comment:
doesn't this turn `in_prox_k = true` if _any_ of the cone proximities are within `prox_bound` when not using sum prox?

Reply:
in_prox_k is only for cone k